tests/dummy/app/locales/en/translations.js

```diff
@@ -417,20 +417,20 @@ Ember.$.extend(true, translations, {
     'components-examples': {
       'flexberry-checkbox': {
         'settings-example': {
-          'caption': 'Components-examples/flexberry-checkbox/settings-example'
+          'caption': 'Flexberry-checkbox. Settings example'
         }
       },
       'flexberry-datepicker': {
         'settings-example': {
-          'caption': 'Components-examples/flexberry-datepicker/settings-example'
+          'caption': 'Flexberry-datepicker. Settings example'
         }
       },
       'flexberry-dropdown': {
         'settings-example': {
-          'caption': 'Components-examples/flexberry-dropdown/settings-example'
+          'caption': 'Flexberry-dropdown. Settings example'
         },
         'conditional-render-example': {
-          'caption': 'Components-examples/flexberry-dropdown/conditional-render-example',
+          'caption': 'Conditional render example',
```

Comment:
Let's stick to a single style: `{{component-name}}. {{example-subject}}`.
Here and hereinafter.